go/teams/teams.go

```diff
@@ -2802,3 +2802,24 @@ func GetUntrustedTeamInfo(mctx libkb.MetaContext, name keybase1.TeamName) (info
 
 	return teamInfo, nil
 }
+
+func GetUntrustedTeamExists(mctx libkb.MetaContext, name keybase1.TeamName) (exists bool, err error) {
+	res, err := mctx.G().API.Get(mctx, libkb.APIArg{
+		Endpoint:    "team/exists",
+		SessionType: libkb.APISessionTypeREQUIRED,
+		Args: libkb.HTTPArgs{
+			"teamName": libkb.S{Val: name.String()},
+		},
+	})
+	if err != nil {
+		return false, err
+	}
+
+	exists, err = res.Body.AtKey("exists").GetBool()
```

Comment:
can you look at for example:
https://github.com/keybase/client/blob/master/go/bot/service_helper.go#L25

on how to make these calls to API + extracting info easier?